sys-process/cpu-x/cpu-x-3.2.4.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg
+
+MY_PN='CPU-X'
+DESCRIPTION='A Free software that gathers information on CPU, motherboard and more'
+HOMEPAGE="https://x0rg.github.io/${PN}"
+SRC_URI="https://github.com/X0rg/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE='GPL-3+'
+SLOT='0'
+KEYWORDS='-* ~amd64'
+IUSE='l10n_cs l10n_fr l10n_id l10n_pl l10n_pt-BR l10n_ru l10n_zh
++gtk +ncurses static-libs
++bandwidth check-update +dmidecode force-libstatgrab portable'
+RESTRICT='primaryuri'
+PATCHES=(
+	"${FILESDIR}/${P}-check-update.patch"
+	"${FILESDIR}/${P}-l10n-support.patch"
+	"${FILESDIR}/${P}-static-libs-fix.patch"
+)
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+DEPEND='
+		gtk? ( >=x11-libs/gtk+-3.12:3 )
```

Comment:
Please sort this list. Unconditional deps before conditionals, and both in alphabetical order. 

Also where's your RDEPEND?